ts/theme/components/Text.ts

```diff
@@ -8,7 +8,13 @@ declare module "native-base" {
   namespace NativeBase {
     interface Text extends TextProperties {
       link?: boolean;
+      item?: boolean;
```

Comment:
what is `item`? it's the text? attribute names should be descriptive, `item` is very vague